packages/appstore/src/assets/svgs/wallets/image-types.tsx

```diff
@@ -0,0 +1,10 @@
+import { WalletsImagesList } from './index';
+
+export type TWalletsImagesListKeys = keyof typeof WalletsImagesList;
+export type TImageTestID = `dt_${TWalletsImagesListKeys}`;
+
+export type WalletsImageProps<T> = {
+    image: T;
+    className?: string;
+    width?: number;
+};
```

Comment:
Should this file be in `packages/appstore/src/assets/svgs` instead of inside `/wallets`? 🤔

Reply:
@nijil-deriv it's the same that we discussed with @hamid-deriv a couple of days ago why component `WalletsImage` is inside of `assets` folder and not inside of `components` folder. But @mahdiyeh-deriv said it's ok. Also we already have `CurrencyIcon` and `TradingPlatformIcon` components in this folder and `icon-types.tsx` file in the same folder as well. So I suppose it's not a problem of my PR, this is a common problem which we need to discuss.

Reply:
I understand that we have `icon-types.tsx` in `packages/appstore/src/assets/svgs`, which is why my question is why not put `image-types.tsx` in the same path instead of inside `packages/appstore/src/assets/svgs/wallets`?

Reply:
@sergei-deriv It's because IconProps from `icon-types.tsx` uses in two places: `CurrencyIcon` and `TradingPlatformIcon`, it's common type for two components, but `image-types.tsx` uses only for `WalletsImage` component. But I understand what you mean and I think you are right, but my suggestion is to make this refactor later when we will stitch components to one (https://app.clickup.com/t/20696747/WALL-167) because current card is in QA right now

Reply:
Ah, okay 👌